app/models/samples/json/schedule_fixedinterval_increasing_fine_timestep.json

```diff
@@ -0,0 +1,158 @@
+{
+    "type": "ScheduleFixedIntervalAbridged",
+    "name": "Solstice Increasing",
+    "values": [
+        0.0,
+        0.007,
+        0.014,
+        0.021,
+        0.028,
+        0.035,
+        0.042,
+        0.049,
+        0.0559,
+        0.0629,
+        0.0699,
+        0.0769,
+        0.0839,
+        0.0909,
+        0.0979,
+        0.1049,
+        0.1119,
+        0.1189,
+        0.1259,
+        0.1329,
+        0.1399,
+        0.1469,
+        0.1538,
+        0.1608,
+        0.1678,
+        0.1748,
+        0.1818,
+        0.1888,
+        0.1958,
+        0.2028,
+        0.2098,
+        0.2168,
+        0.2238,
+        0.2308,
+        0.2378,
+        0.2448,
+        0.2517,
+        0.2587,
+        0.2657,
+        0.2727,
+        0.2797,
+        0.2867,
+        0.2937,
+        0.3007,
+        0.3077,
+        0.3147,
+        0.3217,
+        0.3287,
+        0.3357,
+        0.3427,
+        0.3497,
+        0.3566,
+        0.3636,
+        0.3706,
+        0.3776,
+        0.3846,
+        0.3916,
+        0.3986,
+        0.4056,
+        0.4126,
+        0.4196,
+        0.4266,
+        0.4336,
+        0.4406,
+        0.4476,
+        0.4545,
+        0.4615,
+        0.4685,
+        0.4755,
+        0.4825,
+        0.4895,
+        0.4965,
+        0.5035,
+        0.5105,
+        0.5175,
+        0.5245,
+        0.5315,
+        0.5385,
+        0.5455,
+        0.5524,
+        0.5594,
+        0.5664,
+        0.5734,
+        0.5804,
+        0.5874,
+        0.5944,
+        0.6014,
+        0.6084,
+        0.6154,
+        0.6224,
+        0.6294,
+        0.6364,
+        0.6434,
+        0.6503,
+        0.6573,
+        0.6643,
+        0.6713,
+        0.6783,
+        0.6853,
+        0.6923,
+        0.6993,
+        0.7063,
+        0.7133,
+        0.7203,
+        0.7273,
+        0.7343,
+        0.7413,
+        0.7483,
+        0.7552,
+        0.7622,
+        0.7692,
+        0.7762,
+        0.7832,
+        0.7902,
+        0.7972,
+        0.8042,
+        0.8112,
+        0.8182,
+        0.8252,
+        0.8322,
+        0.8392,
+        0.8462,
+        0.8531,
+        0.8601,
+        0.8671,
+        0.8741,
+        0.8811,
+        0.8881,
+        0.8951,
+        0.9021,
+        0.9091,
+        0.9161,
+        0.9231,
+        0.9301,
+        0.9371,
+        0.9441,
+        0.951,
+        0.958,
+        0.965,
+        0.972,
+        0.979,
+        0.986,
+        0.993,
+        1.0
+    ],
+    "timestep": 6,
```

Comment:
Should there be a default value for timestep?

Reply:
Yes, the default value should be 1 since I imagine most `ScheduleFixedIntervals` will use 1 value per hour.  The key is there in the event that someone wants to use a higher timestep.